packages/s2-core/src/data-set/custom-tree-pivot-data-set.ts

```diff
@@ -94,6 +92,11 @@ export class CustomTreePivotDataSet extends PivotDataSet {
       ...restCfg,
       meta: newMeta,
       data: uniq(transformedData),
+      fields: {
```

Comment:
上面那个 `valueInCols` 也放进来吧

```suggestion
     fields: {
        ...dataCfg.fields,
        // 1、rows 配置必须是空，需要额外添加 $$extra$$ 定位数据（标记指标的id）
        rows: [EXTRA_FIELD],
        valueInCols: false
      },
```